handler/date/date.go

```diff
@@ -1,17 +1,18 @@
 package date
 import(
 	"fmt"
+	"strconv"
 	"github.com/gin-gonic/gin"
 	"Schedule/service/token"
 	"Schedule/model"
 	"Schedule/handler/user"
 )
-var Date model.Date
+var Date model.Date//全局变量date
 
 func SearchDate(c *gin.Context){
-	Date.Year =c.PostForm("year")
-	Date.Month=c.PostForm("month")
-	Date.Day=c.PostForm("day")
+	Date.Year,_=strconv.Atoi(c.PostForm("year"))

```

Comment:
This error cannot be ignored.

`Data.Year` will be nil if someone pass an invalid int parameter.